docs/user-guide/zowe-getting-started-tutorial.md

```diff
@@ -2,20 +2,23 @@
 
 Learn how to start exploring the Zowe components, applications and plug-ins.
 
-**Use core components:**
+## Use Zowe server-side components
```

Comment:
I'd suggest removing all instances of "Use" in the headers b/c most of the bullets below already start w/ "Using," so it feels redundant to have the same verb both in the header and the bullets.